File: app/src/main/java/me/echeung/moemoekyun/utils/PluralsUtil.java
package me.echeung.moemoekyun.utils;

import android.content.Context;

public final class PluralsUtil {

    public static String getString(Context context, int pluralId, int value) {
        String text = context.getResources().getQuantityString(pluralId, value);
        return String.format(text, value);
    }

}

File: app/src/main/java/me/echeung/moemoekyun/util/system/PluralsUtil.kt
package me.echeung.moemoekyun.util.system

import android.content.Context

object PluralsUtil {

    @JvmStatic
    fun getString(context: Context, pluralId: Int, value: Int): String {
        val text = context.resources.getQuantityString(pluralId, value)
        return String.format(text, value)
    }

}
